### DISCUSSION TITLE
Extracting data from tables

### DISCUSSION THREAD

**Discussion by Kamil-Braschke** [haotian-liu/LLaVA#666] [Q&A]
Hello!

Has anybody experience in extracting data from tables using LLaVA? I tried it in the demo section and it failed miserably - it was not able to do anything correctly and regularly got stuck in a loop repeating itself. 
If anyone has some suggestions as to how to tackle this, I would gladly hear them.

**Comment by abhirupghosh** [comment]
It surprisingly performed pretty well for my requirements. Is there anything specific that went wrong for your use case?

**Comment by Kamil-Braschke** [comment]
Well, it is quiet some time ago, so the newest version might perform better. I do not recall the details anymore, but like I said in my original post, at that time it was not able to extract the asked information from a table and acted buggy (repeating itself).
